src/components/DisplayColours.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+function DisplayColours({ colors }) {
+  return (
+    <>
+    <h3>Thank you for choosing Color Compass</h3>
+    {colors.map(color => (
+        <div key={color.hex}>
+            <div className={`bg-${color.hex.slice(1)}-300 rounded-full p-6 inline-block`}
+            style={{ backgroundColor: color.hex }}></div>
+        <div className={`bg-${color.hex.slice(1)}-300 rounded-full inline-block ml-4 p-4`}>
+        <p>Hex: {color.hex}z</p>
+        <p>Name: {color.name}</p>
+    <div>
+        <p>Description: {color.description}</p>
```

Comment:
Line 9: Tailwind CSS doesn't support dynamic class names:
{`bg-${color.hex.slice(1)} this doesn't work properly, style is working.
Line 11: why do we need it?
Line:12/13/15 are made not according to Figma design

Reply:
Ok, any suggestions on how to change those last lines so they are according to Figma design? I took out the color.hex.slice part so will push those changes up, also deleted line 11